### PR TITLE
Mobilify

### PR DESCRIPTION
**Pull request by jennazee**
I wanted to make some things smaller for mobile...

Wanted to do more, i.e. minifying css, but ran out of time :(

### CODE REVIEW

**Comment by chrisabrams** [comment]
This PR deleted the map of the museum.

File: views/layouts/main.hbs [hunk 1]
```diff
@@ -25,50 +35,6 @@
     ga('create', 'UA-72369471-1', gaTrackerOptions);
     ga('send', 'pageview');
   </script>
-  <script src="//maps.googleapis.com/maps/api/js?sensor=false"></script>
```
Comment (chrisabrams) [inline views/layouts/main.hbs:28]: here - this is the code for the map

Comment (jennazee) [inline views/layouts/main.hbs:28]: I don't see the map on the prod version so I thought it was leftovers

Comment (chrisabrams) [inline views/layouts/main.hbs:28]: Yep - same place: https://github.com/EmpireJS/2016.empirejs.org/blob/dcc1dc7d355226e88aab9bf39ad20fd09936530f/views/layouts/main.hbs#L28

Comment (jennazee) [inline views/layouts/main.hbs:28]: I meant if you go to 2016.empirejs.org, do you see the map?

Comment (jennazee) [inline views/layouts/main.hbs:28]: Oh shit i was in mobile and it goes away. Will restore now. Thanks!

**Comment by jennazee** [comment]
At this point, the maps impact the load time more than anything else, so I'm ambivalent about getting this in. Load times are anywhere from 2-8s on my home wifi, so I'm curious what phones will do.

**Comment by hackygolucky** [comment]
Looks great to me! In mobile, dropping the map seems fine. I say ship it.
Let me know if you need help deploying.

**Comment by jennazee** [comment]
Maps js gets loaded regardless, it just doesn't show up on mobile, which isn't my fave but shrug.

**Pull request merged**